docs/explanation/intro-to-ontologies.md

```diff
@@ -261,6 +261,19 @@ If we then have an entity `nose` that is subClassOf `sensory organ` and `capable
 
 ![](../images/discussions/intro-to-ontologies/nose-classification.png)
 
+##### How to interpret an 'and' or an 'or'
+
+In the case where a class has 'and' in the label, such as MONDO:0017411 'neonatal inflammatory skin and bowel disease', this should be interpreted as someone with this disease has both neonatal inflammatory skin disease and bowel disease at the same time. This class should be classified as a child of 'bowel disease' and 'neonatal inflammatory skin disease'. 
+
+Having different statements in the SubClass Of means that the instance of the class is a combination of all the SubClass Of statements. For example, if 'neonatal inflammatory skin and bowel disease' is a subclass of 'bowel disease' and 'neonatal inflammatory skin disease', then an individual with this disease has 'bowel disease' and 'neonatal inflammatory skin disease'.
+
+<img width="424" alt="image" src="https://user-images.githubusercontent.com/6722114/220412425-3585f922-f1db-4d77-8ee3-48350ff5f55c.png">
+
+If there were a class called 'neonatal inflammatory skin **or** bowel disease', then it would be interpreted that an individual with this disease has _either_ bowel disease _or_ neonatal inflammatory skin disease _or_ both. It would not be accurate to classify this class as a child of bowel disease and neonatal inflammatory skin disease.
```

Comment:
```suggestion
If there were a class 'neonatal inflammatory skin **or** bowel disease', the intention is usually that this class follows disjunctive logic. A class following this logic would be interpreted in a way that an individual with this disease has _either_ bowel disease _or_ neonatal inflammatory skin disease _or_ both. It would not be accurate to classify this class as a child of bowel disease and neonatal inflammatory skin disease. This type of class is often called a "grouping class", and is used to aggregate related diseases in a way useful to users, like "disease" and "sequelae of disease".
```